pandas/tests/extension/json/test_json.py

```diff
@@ -322,20 +329,23 @@ def test_groupby_extension_apply(self):
         I suspect that once we support Index[ExtensionArray],
```

Comment:
@WillAyd we now support Index[EA] so this comment may now be addressable (im just guessing you wrote it)